src/Functions/formatDateTime.cpp

```diff
@@ -873,6 +915,22 @@ class FunctionFormatDateTimeImpl : public IFunction
 
                 switch (*pos)
                 {
+                    // Abbreviated weekday [Mon...Sun]
+                    case 'a':
+                        instructions.emplace_back(&Action<T>::mysqlDayOfWeekTextShort);
+                        break;
+
+                    // Abbreviated month [Jan...Dec]
+                    case 'b':
+                        instructions.emplace_back(&Action<T>::mysqlMonthOfYearTextShort);
+                        break;
+
+                    // Month as a decimal number (01-12)
```

Comment:
[MySQL docs](https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_date-format) state for `c`:
```
Month, numeric (0..12)
```

Do you this is only a typo or are there any situations where the output can actually be "0"? Asking because in case it is not a typo, we might want to either replicate their behavior or at least think about documenting the difference.

Reply:
That is a good question. I was copying the actual [mysql docs](https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_date-format) (which states `00-12`), however, doesn't state which side is exclusive.

Playing around with a mysql database displays January as `01` so it should be read as `(00-12]` or 01-12.

Reply:
That makes sense, thanks for checking.